codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/PaginatorGenerator.kt

```diff
@@ -124,6 +116,12 @@ class PaginatorGenerator : KotlinIntegration {
         """.trimIndent()
         val docReturn = "@return A [kotlinx.coroutines.flow.Flow] that can collect [${outputSymbol.name}]"
 
+        val inputParameter = if (paginationInfo.input.hasAllOptionalMembers) {
+            writer.format("initialRequest: #1T = #1T{}", inputSymbol)
```

Comment:
nit (style): `#1T { }`

```kt
initialRequest: PutObjectRequest = PutObjectRequest{}

vs.

initialRequest: PutObjectRequest = PutObjectRequest { }
```

If you think it's a good suggestion, also applies to the same change in waiters